src/Game.js

```diff
@@ -0,0 +1,61 @@
+import Car from './Car.js';
+import RandomSingleDigitGenerator from './utils/RandomSingleDigitGenerator.js';
+import OutputManager from './OutputManager.js';
+
+class Game {
+  constructor(carNames, attempts) {
+    this.cars = this.createCars(carNames);
+    this.attempts = attempts;
+    this.randomSingleDigitGenerator = new RandomSingleDigitGenerator();
+  }
+
+  createCars(carNames) {
```

Comment:
```suggestion
  #createCars(carNames) {
```

클래스 내부에서만 사용되는 로직은 # 키워드를 통해 `private`으로 만들어주면 좋을 거 같습니다!